README.md

```diff
@@ -342,22 +342,22 @@ To contribute to uutils, please see [CONTRIBUTING](CONTRIBUTING.md).
 | Done      | Semi-Done | To Do  |
 |-----------|-----------|--------|
 | arch      | cp        | chcon  |
-| base32    | expr      | csplit |
-| base64    | install   | dd     |
-| basename  | ls        | df     |
-| cat       | more      | numfmt |
-| chgrp     | od (`--strings` and 128-bit data types missing) | runcon |
-| chmod     | printf    | stty   |
-| chown     | sort      |        |
-| chroot    | split     |        |
-| cksum     | tail      |        |
-| comm      | test      |        |
-| csplit    | date      |        |
-| cut       | join      |        |
-| dircolors | df        |        |
+| base32    | date      | dd     |
+| base64    | df        | runcon |
+| basename  | expr      | stty   |
+| cat       | install   |        |
+| chgrp     | join      |        |
+| chmod     | ls        |        |
+| chown     | more      |        |
+| chroot    | numfmt    |        |
```

Comment:
I'm not 100% sure if `numfmt` fits more in Semi-Done or Done. Let me know, if I should move it to "Done" and I'll do that.

Reply:
Comparing `uu_numfmt` with the GNU `numfmt` manpage, I think you made the right call!